### PR TITLE
Added missing fields in products and orders

### PR DESCRIPTION
**Pull request by RanAbram**
https://help.shopify.com/api/reference/order
https://help.shopify.com/api/reference/product

@dlebech

### CODE REVIEW

File: order.go [hunk 1]
```diff
@@ -137,8 +145,8 @@ type LineItem struct {
 	Grams                      int              `json:"grams"`
 	FulfillmentStatus          string           `json:"fulfillment_status"`
 	TaxLines                   []TaxLine        `json:"tax_lines"`
-	OriginLocation             Address          `json:"origin_location"`
-	DestinationLocation        Address          `json:"destination_location"`
+	OriginLocation             *Address         `json:"origin_location"`
+	DestinationLocation        *Address         `json:"destination_location"`
```
Comment (dlebech) [inline order.go:149]: This is potentially a breaking change for some users. Have you noticed any issues with this *not* being a pointer?

**Review by dlebech — COMMENTED** [review]
Thanks for the PR @RanAbram and sorry for the wait! I just have a small question to one of the changes, otherwise it looks good to me.

File: order.go (hunk 1 above)
Comment (RanAbram) [reply order.go:149]: Well i didn't have any issue with this being not a pointer..
I think the right way in Go is to use it as a pointer, and i also tried to changed it into pointer in order to stick to the convention the `Address` properties are being used inside the `Order` struct.

@dlebech  From my side I really don't mind to use it the way it was before. Let me know if you wish to keep using it as it was.

Comment (dlebech) [reply order.go:149]: @RanAbram I think that makes sense, it's also easier to check if the object is "empty" by checking for nil. Thanks for sharing your thoughts.

**Review by dlebech — APPROVED** [review]

**Pull request merged**